thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -114,8 +117,9 @@ private void runTask(DateTime windowStart, DateTime windowEnd)
     boolean isBackfill = false;
     // If the current job is a backfill (adhoc) detection job, set notified flag to true so the merged anomalies do not
     // induce alerts and emails.
-    String jobName = DAO_REGISTRY.getJobDAO().getJobNameByJobId(jobExecutionId);
-    if (jobName != null && jobName.toLowerCase().startsWith(BACKFILL_PREFIX)) {
+    if (detectionJobType != null && (detectionJobType.equals(DetectionJobContext.DetectionJobType.BACKFILL) ||
```

Comment:
Try to follow convention of importing upto DetectionJobContext and then using DetectionJobType directly, for sake of consistency with rest of the code (same comment for all usages of DetectionJobContext.DetectionJobType)